docs/sql-guide/functions/function-euclidean-distance.md

```diff
@@ -0,0 +1,32 @@
+---
+title: EUCLIDEAN_DISTANCE()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+# EUCLIDEAN_DISTANCE() function
+
+`EUCLIDEAN_DISTANCE()` is a mathematical function that can be performed on `VECTOR` data type columns.
+The Euclidean (or Euclidian) distance between two vectors is the square root of the sum of the squares
```

Comment:
> The Euclidean (or Euclidian) distance between two vectors is the square root of the sum of the squares the pairwise differences between the vectors. This is the distance metric most commonly used when talking about distances between locations on a plane.

Are these the same explanation repeated, or are there missing words between?
* "square root of the sum of the squares"
* "the pairwise differences between the vectors"

And for that matter, what's a "pairwise difference"? 

Note: something I personally find irritating in tech (not just here, so please don't think I'm aiming at you), are explanations that drop in terminology that's not clearly explained. I just tried searching on "pairwise difference" on Ze InterTubes and found it mentioned in LOTS of places, but NEVER ACTUALLY EXPLAINED! ARGH!

Suggested rewrite here:

```
Euclidean distance is commonly used to measure distance between locations on a plane. 
The function It is calculated by adding the squares of the pairwise differences between between two vectors then taking the square root.
```